test_scripts/Capabilities/PersistingHMICapabilities/common.lua

```diff
@@ -118,22 +118,30 @@ function m.checkContentOfCapabilityCacheFile(pExpHmiCapabilities)
       for req, params in pairs(requests) do
         for _, param in ipairs(params) do
           local message = mod .. "." .. param
-          if param == "audioPassThruCapabilitiesList" then
-            if not utils.isTableEqual(cacheTable[mod].audioPassThruCapabilities, expHmiCapabilities[mod][req].params[param]) then
-              errorMessages = errorMessages ..
-                errorMessage(message, cacheTable[mod].audioPassThruCapabilities,
-                  expHmiCapabilities[mod][req].params[param])
-            end
-          else
-            if not cacheTable[mod][param] then
-              errorMessages = errorMessages ..
-                errorMessage(message, "does not exist", expHmiCapabilities[mod][req].params[param])
-            else
-              if not utils.isTableEqual(cacheTable[mod][param], expHmiCapabilities[mod][req].params[param]) then
+          if not cacheTable[mod][param] then
+            errorMessages = errorMessages ..
+              errorMessage(message, "does not exist", expHmiCapabilities[mod][req].params[param])
+          elseif param == "audioPassThruCapabilitiesList" then
+            if not utils.isTableEqual(cacheTable[mod].audioPassThruCapabilities,
+              expHmiCapabilities[mod][req].params[param]) then
                 errorMessages = errorMessages ..
-                  errorMessage(message, cacheTable[mod][param], expHmiCapabilities[mod][req].params[param])
+                  errorMessage(message, cacheTable[mod].audioPassThruCapabilities,
+                    expHmiCapabilities[mod][req].params[param])
+            end
+          elseif param == "remoteControlCapability" then
+            for _, buttonCap in ipairs(expHmiCapabilities[mod][req].params[param].buttonCapabilities) do
+              if buttonCap.moduleInfo.allowMultipleAccess == nil then
+                buttonCap.moduleInfo.allowMultipleAccess = true
               end
             end
+            if not utils.isTableEqual(cacheTable[mod][param], expHmiCapabilities[mod][req].params[param]) then
+              errorMessages = errorMessages ..
+                errorMessage(message, cacheTable[mod][param],
+                  expHmiCapabilities[mod][req].params[param])
+            end
+          elseif not utils.isTableEqual(cacheTable[mod][param], expHmiCapabilities[mod][req].params[param]) then
```

Comment:
@VjKlepikov As I can see this snap of code is repeated quite often in  current function.
```
if not utils.isTableEqual(cacheTable[mod][param], expHmiCapabilities[mod][req].params[param]) then
              errorMessages = errorMessages ..
                errorMessage(message, cacheTable[mod][param],
                  expHmiCapabilities[mod][req].params[param])
            end
```
Could it be placed in separate inner local function and replaced by call of this function in main body of current function?